src/game_io/game_class_packet.cc

```diff
@@ -27,7 +27,7 @@ namespace Widelands {
 
 constexpr uint16_t kCurrentPacketVersion = 6;
 
-void GameClassPacket::read(FileSystem& fs, Game& game, MapObjectLoader*) {
+void GameClassPacket::read(FileSystem& fs, Game& game, MapObjectLoader* /*unused*/) {
```

Comment:
as this is declared as nullptr in the header file would it be safe to delete it completely?

Reply:
We can't delete the parameter from the function signature because this method is overridden from the parent class `GameDataPacket`. I changed all occurrences of `/*unused*/` to meaningful names in this and the other PRs now though.